client/hbs/editor.hbs

```diff
@@ -34,9 +34,18 @@
 				</button>
 			</footer>
 		</main>
+		<div class='uploader'>
+			<button id='uploadTrigger' class='upload-button upload-item'>Upload New File</button>
+			<div class='upload-name upload-item disabled'></div>
+			<a href='/files/' target='_blank' class='button upload-launch upload-item'>
+				Browse <i class="material-icons">launch</i>
```

Comment:
`<i>` is not the semantically correct choice here.

Also, `<dttw-icon>` should work? :)